src/poetry/repositories/repository_pool.py

```diff
@@ -77,7 +85,9 @@ def repository(self, name: str) -> Repository:
         raise IndexError(f'Repository "{name}" does not exist.')
 
     def add_repository(
-        self, repository: Repository, default: bool = False, secondary: bool = False
+        self,
+        repository: Repository,
+        priority: Priority = Priority.PRIMARY,
```

Comment:
If I remember correctly, this function is used by plugin authors (not only poetry-plugin-export). Maybe, we should deprecate the parameters, keep them functional and add `priority` as keyword-only argument for now.